src/app/calendar-container/calendar-container.component.scss

```diff
@@ -145,6 +145,9 @@ router-outlet.router-flex + * {
 /* mobile css */
 @media screen and (max-width: 768px) {
     /* display short day abbreviations */
+    #calendar-controls-wrapper {
```

Comment:
we will also normalize all max-width breakpoints to something that makes sense rather than 768px at a later date